torchbenchmark/models/fambench_dlrm/config.py

```diff
@@ -52,10 +52,10 @@ class FAMBenchEvalConfig:
     test_mini_batch_size:int = 1024
     test_num_workers:int = 0
     data_generation:str = "random"
-    arch_mlp_bot:str = "1414-1750-1750-1750-1750-1750-1750-1750-1750-96"
-    arch_mlp_top:str = "1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1"
+    arch_mlp_bot:str = "1414-1750-1750-1750-1750-1750-1750-96"
+    arch_mlp_top:str = "1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1450-1"
     arch_sparse_feature_size:int = 96
-    arch_embedding_size:str = "555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693"
+    arch_embedding_size:str = "555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693-555693"
```

Comment:
Why are the modifications here?

Reply:
this is dlrm embedding layer size,original config make it too large to fill in A10 GPU.